api.md

```diff
@@ -1832,29 +1832,75 @@ DROP MATERIALIZED VIEW contagg_view;
 
 ## refresh_continuous_aggregate() :community_function: [](refresh_continuous_aggregate)
 
-Refresh all buckets of a continuous aggregate between two points of
-time. 
+Refresh all buckets of a continuous aggregate in the _refresh window_
+given by `window_start` and `window_end`.
+
+A continuous aggregate materializes aggregates in time buckets (e.g.,
+min, max, average over 10 minutes of data), as determined by the
+continuous aggregate view query's `time_bucket` expression. Therefore,
+when refreshing the continuous aggregate, only buckets that completely
+fit within the refresh window will be refreshed. In other words, it is
+not possible to compute the aggregate over, e.g., half a bucket, and
```

Comment:
I usually avoid "e.g." and "i.e." in more formal writing, such as manuals and books, and instead use "for example" and "that is". It is easier on the eye.
```suggestion
not possible to compute the aggregate over, for example, half a bucket, and
```